File: c_Programming/Assingements/Assignment 7/find_sum.c
#include<stdio.h>
void find_sum(int*);
void main()
{
	int no;
	printf(" Enter a number :");
	scanf("%d",&no);
	find_sum(&no);
}
void find_sum(int*no)
{
	int f,l,sum;
	l=*no%10;
	while(*no>1)
	{
	  *no=*no/10;
	}
	f=*no;
	sum=f+l;
	printf("%d (%d + %d) ", sum,f,l);
}

File: Java_Module/Assignment/Assignment6/ArrayProgram/src/FindSum.java
import java.util.Scanner;
class FindSum {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int[]arr=new int[10];
		
		//Input Array
		System.out.println("Enter a 10 element : ");
		for(int i=0; i<arr.length; i++) {
			arr[i]=sc.nextInt(10);
		}
		//Display Array
		System.out.print("Array = [");
		for(int i=0; i<arr.length; i++) {
			System.out.print(arr[i] + " ");
		}
		System.out.print("]");
		System.out.println();
		
		//find the sum of Array
		int sum=0;
		for(int i=0; i<arr.length; i++) {
			sum=sum+arr[i];
		}
		System.out.println("sum = "+sum);
		sc.close();
	}

}
